doc/api/globals.md

```diff
@@ -1075,6 +1075,17 @@ added: v17.0.0
 
 The WHATWG `DOMException` class. See [`DOMException`][] for more details.
 
+## `Temporal`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental.
+
+The `Temporal` global is a new language level API for working with dates and
+times.
+
```

Comment:
I don't think it makes sense to list it here, since it's not our implementation, it is a built-in object (we do not list `JSON` or `Atomics` either)

https://github.com/nodejs/node/blob/d1f8ccb10daba0b4eb4db4986615a545e11a0d87/doc/api/globals.md#L18-L20

Reply:
Yep, I wasn't sure how else to indicate that it would still be experimental tho. We don't really have another place to list it.

Reply:
Yeah I guess we don't really have a precedent for an experimental global that's not implemented by us. What do we think about having an experimental runtime warning instead?

Reply:
A warning is certainly a possibility here. I'm not sure if that should be in replacement for the documentation tho. Will have to think about it.

Reply:
How about mentioning the flag (or its inverse, depending on how this is going to be exposed) under `cli.md#useful-v8-options`?